### PR TITLE
Update state API in Endpoint Specifications

### PR DESCRIPTION
**Pull request by rranjan3**
Update /state and /state/{address} API references. Add
detail for partial address support.

Signed-off-by: Rajeev Ranjan <rajeev2.ranjan@intel.com>

### CODE REVIEW

**Comment by kulkarniamol** [comment]
Looks ok to me. 
@chenette pls review and approve.

**Comment by chenette** [comment]
@kulkarniamol - I'm on vacation, so I'm hoping that @delventhalz and others can handle this PR.

File: rest_api/openapi.yaml [hunk 1]
```diff
@@ -200,6 +200,10 @@ paths:
         to a particular head block. Using the `address` filter parameter will
         narrow the list to any entries that have an address beginning with the
         characters specified.
+
+        Note that the partial address in `address` parameter should have even 
+        number of hecadeximal characters(i.e., complete bytes).
```
Comment (delventhalz) [inline rest_api/openapi.yaml]: Should be a space before the open parentheses.

**Comment by delventhalz** [comment]
@rranjan3 I'm approving the PR because it looks good overall, but there is a typo you should fix before merging.

**Pull request merged**